components/calendar/src/date.rs

```diff
@@ -197,6 +198,7 @@ impl<A: AsCalendar> Date<A> {
     /// Construct a date from a [`RataDie`] and some calendar representation
     #[inline]
     pub fn from_rata_die(rd: RataDie, calendar: A) -> Self {
+        let rd = rd.clamp(*VALID_RD_RANGE.start(), *VALID_RD_RANGE.end());
```

Comment:
suggestion: mention in docs?

we _probably_ should document the general validity range somewhere, too.